apps/mobile/src/features/send/send-form/send-form.schema.ts

```diff
@@ -0,0 +1,19 @@
+import { z } from 'zod';
+
+export const sendFormSchema = z.object({
+  amount: z.string().refine(val => !isNaN(Number(val)) && Number(val) > 0, {
+    message: '',
+  }),
+  recipient: z.string(),
+  memo: z.string().optional(),
+  fee: z.string(),
+});
```

Comment:
Parts of this will certainly need to be modified and composed for different chains/tokens. How might we go about doing that?